src/inputs/Select/Select.js

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React, { useMemo, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { Stack, Autocomplete, TextField } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+import { TooltipInfo } from '../../misc';
+import { BasicInputPlaceholder } from '../BasicInputs/BasicInputPlaceholder';
+import { getCommonInputStyles } from '../style';
+
+const useStyles = makeStyles(getCommonInputStyles);
+
+export const Select = (props) => {
```

Comment:
I don't know if it was discussed earlier but just _Select_ for the name seems a bit too large :thinking: we have `MultiSelect`, `SelectWithAction`, `BasicEnumInput` and now, a fourth component that does almost the same thing, would it be possible to find something more specific to distinguish it from its three siblings ?

Reply:
Yes, we discussed with THU and for this moment, we will let the component name as Select but you're right : I also think it is bit too large.

Reply:
Maybe `Autocomplete` or `SingleSelect` ?
(it feels like `MultiSelect` and `SingleSelect` should be a single component with an option though)

Reply:
Yes, I'd say that `Select` component could be the one that gives both opportunities with an option but for now, we can just separate both in `MultiSelect` and `SingleSelect` :+1:

Reply:
Coincidence, at the beginning, I thought about _SingleSelect_...